src/styles/menu-slider.js

```diff
@@ -76,7 +84,7 @@ export default css`
     bottom: 0;
     left: ${menuButtonWidth};
     z-index: 1;
-    transform: translateX(calc(var(--menuWidth) * -1));
+    transform: translateX(calc(100% * -1));
```

Comment:
Do you need this `calc()` still since it's a fixed value?

Reply:
yah, it has do find this number: `100% * -1`